### PR TITLE
fix: verify checksums of downloaded kernel packages

### PR DESCRIPTION
**Pull request by wizard-28**

### CODE REVIEW

**Comment by ajstrongdev** [comment]
I have added myself and @jbrgilbrech as a reviewer, I will review this PR in-depth when I have the free time either tomorrow or Wednesday. In the meantime @jbrgilbrech is free to review the PR.

**Comment by ajstrongdev** [comment]
I will review the PR once I have merged #36

File: update.sh [hunk 1]
```diff
@@ -31,16 +31,21 @@ sudo mv rhino-config /usr/bin
...
+    
+    echo "Verifying checksums..."
+    if shasum --check CHECKSUMS; then
+      sudo apt install ./*.deb
+      : > "$HOME/.rhino/config/5-17-5"
+    else
+      >&2 echo "Failed to verify checksums of downloaded kernel files!"
+      exit 1
+    fi
 fi
```
Comment (ajstrongdev) [inline update.sh]: Everything looks good.

**Review by ajstrongdev — APPROVED** [review]
Looks good, I will test it later on tomorrow. I love the fact that you made the if statements better by placing them on the same line.

**Comment by ajstrongdev** [comment]
@cat-master21 A lot of changes have happened to rhino-update and there are conflicts that must be resolved. Can you resolve them and use the upstream changes? Thank you.

File: update.sh [hunk 2]
```diff
@@ -31,16 +31,22 @@ sudo mv rhino-config /usr/bin
 rm -rf ~/rhino-config
 
 # If the user has selected the option to install the mainline kernel, install it onto the system.
-if [[ -f "$HOME/.rhino/config/mainline" ]]; then
-  if [[ ! -f "$HOME/.rhino/config/5-17-9" ]]; then
+if [[ -f "$HOME/.rhino/config/mainline" ]] && [[ ! -f "$HOME/.rhino/config/5-17-9" ]]; then
     cd ~/rhinoupdate/kernel/
+    wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.17.9/amd64/CHECKSUMS
```
Comment (ajstrongdev) [inline update.sh:36]: Will the checksums url have to be updated every time a new kernel is released?

**Review by ajstrongdev — APPROVED** [review]

File: update.sh (hunk 2 above)
Comment (wizard-28) [reply update.sh:36]: Obviously?

Comment (ajstrongdev) [reply update.sh:36]: Okay just confirming

**Comment by ajstrongdev** [comment]
Testing today.

**Pull request merged**